czkawka_cli/src/commands.rs

```diff
@@ -197,16 +197,16 @@ pub struct SimilarImagesArgs {
         value_parser = parse_minimal_file_size,
         default_value = "18446744073709551615",
         help = "Maximum size in bytes",
-        long_help = "Maximum size of checked files in bytes, assigning lower value may speed up searching"
+        long_help = "Maximum size of checked files in bytes; assigning lower values may speed up searching"
     )]
     pub maximal_file_size: u64,
     #[clap(
         short,
         long,
         default_value = "High",
         value_parser = parse_similar_images_similarity,
-        help = "Similairty level (Minimal, VerySmall, Small, Medium, High, VeryHigh, Original)",
```

Comment:
looks   Original is missed?

Reply:
Original is not supported as a value that can be passed:

https://github.com/qarmin/czkawka/blob/2a32a52aa882f6ff52ba4d3e24a666dc2a86cf9b/czkawka_cli/src/commands.rs#L575-L582

I was actually planning to propose adding it as an option (perhaps renaming it to "Maximal" to make it more accurate) in a separate PR :)